src/Component/UserBox.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import styled from 'styled-components';
+//import { useMessageState, useMessageDispatch } from '../Context/MessageContext';
+
+
+const UserBox = ( ) => {
+    //const state = useMessageState();
+    //const dispatch = useMessageDispatch();
+
+    const onClick = ( e : React.FormEvent ): void => {
+        e.preventDefault(); //여기에 current 유저 토글기능 넣기
+    }
+
+    return<>
+        <UserField onClick={onClick}>
+            <UserProfile>
+                <ProfileImg/>
```

Comment:
이런 식으로 작성하시면 이미지를 사용하실 수 있습니다.

```suggestion
                <ProfileImg src={require(`../0.png`)}/>
```